code/style.css

```diff
@@ -1,7 +1,129 @@
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+html {
+  background: #e1c3e7;
+  color: pink;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  }
```

Comment:
If you want your hero image to go all the way to the edges, you can give the `body` a margin of 0.